src/apps/tools/OswAppTimeConfig.cpp

```diff
@@ -180,9 +180,21 @@ void OswAppTimeConfig::loop() {
         if (OswServiceAllTasks::wifi.isConnected()) {
             OswUI::getInstance()->setTextCursor(BUTTON_2);
             hal->gfx()->print(LANG_TFW_UPDATE);
+            // ** if time is still being updated reprint this status on the display
+            if (OswServiceAllTasks::wifi.getNTPstatus()  ) {    
+                hal->gfx()->setTextCursor(210,150);
+                hal->gfx()->print(LANG_NTP_QUEUED);  // ** NTP Updating message
+            }
             if (hal->btnHasGoneDown(BUTTON_2)) {
                 if (OswServiceAllTasks::wifi.isConnected()) {
                     OswServiceAllTasks::wifi.queueTimeUpdateViaNTP();
+                    // ** when time update button2 is pressed, display NTP updating message
+                    // ** until update is complete
+                    hal->gfx()->setTextCursor(210,150);
+                    hal->gfx()->print(LANG_NTP_QUEUED);  // ** NTP Updating message
+                    #ifndef NDEBUG
+                        Serial.println("updating time via NTP");
+                    #endif
```

Comment:
Please remove it.

Reply:
you want the message removed?

Reply:
remove code.

Reply:
ok, it should still work without that code, I will have to test it to see, then I will remove it if that is the case. good call I think, although the user will not get any indication when they actually press the button, only when the update is still in progress. If you prefer this, I would appreciate if you would confirm.

Reply:
I also have a fairly large update for the watchfaces, wondering if you have any comment on how to submit those changes?